pkg/cmd/pulumi/convert.go

```diff
@@ -100,7 +100,15 @@ func newConvertCmd() *cobra.Command {
 		Short: "Convert Pulumi programs from a supported source program into other supported languages",
 		Long: "Convert Pulumi programs from a supported source program into other supported languages.\n" +
 			"\n" +
-			"The source program to convert will default to the current working directory.\n",
+			"The source program to convert will default to the current working directory.\n" +
+			"\n" +
+			"Valid source languages: yaml, terraform, bicep, arm\n" +
+			"\n" +
+			"Valid target languages: typescript, python, dotnet, go, java, yaml" +
```

Comment:
Shouldn't this be `csharp` (as there are presumably other .NET languages that are, or could be, supported)?

```suggestion
			"Valid target languages: typescript, python, csharp, go, java, yaml" +
```

Reply:
Either works, but csharp is probably better given we've also used typescript not node in this list.